### PR TITLE
Implement peer reviews

### PR DESCRIPTION
**Pull request by irenenikk**

### CODE REVIEW

File: spec/models/peer_review_question_spec.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PeerReviewQuestion, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (vilmakoo) [inline spec/models/peer_review_question_spec.rb]: Would be nice if there weren't any pending specs. Either remove these or write tests?

File: spec/models/peer_review_spec.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PeerReview, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (vilmakoo) [inline spec/models/peer_review_spec.rb]: Pending

File: spec/models/peer_review_question_answer_spec.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PeerReviewQuestionAnswer, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (vilmakoo) [inline spec/models/peer_review_question_answer_spec.rb]: Pending

File: spec/requests/peer_review_question_answers_spec.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'PeerReviewQuestionAnswers', type: :request do
+  describe 'GET /peer_review_question_answers' do
+    it 'works! (now write some real specs)' do
```
Comment (pirtnea) [inline spec/requests/peer_review_question_answers_spec.rb]: Should we address our need for these so called real specs?

File: spec/requests/peer_review_questions_spec.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'PeerReviewQuestions', type: :request do
+  describe 'GET /peer_review_questions' do
+    it 'works! (now write some real specs)' do
```
Comment (pirtnea) [inline spec/requests/peer_review_questions_spec.rb]: These specs are not real.

File: spec/requests/peer_reviews_spec.rb [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'PeerReviews', type: :request do
+  describe 'GET /peer_reviews' do
+    it 'works! (now write some real specs)' do
```
Comment (pirtnea) [inline spec/requests/peer_reviews_spec.rb]: Real specs needed.

**Pull request merged**